src/styles/card.scss

```diff
@@ -0,0 +1,119 @@
+body {
+  margin: 0;
+  background-color: rgb(116, 116, 116);
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  background-color: #fff;
+  width: 200px;
+  height: 408px;
+
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: solid #f3f3f3 1px;
+}
+
+.cardTop {
+  display: flex;
+
+  flex-direction: column;
+  align-items: center;
+
+  &__Img {
+    background-size: cover;
+    background-image: url(../images/imac.jpeg);
+
+    margin-right: 0 19px;
```

Comment:
The `margin-right` property is incorrectly defined as `margin-right: 0 19px;`. This is not a valid CSS syntax. If you intend to set both top and right margins, you should use `margin: 0 19px;`. If you only want to set the right margin, use `margin-right: 19px;`.